new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+
+---> There are a some words fun that don't belong paper in this sentence. {expect:There are some words that don't belong in this sentence.}
+---> There are a some words that don't belong in this sentence. {expect:There are some words that don't belong in this sentence.}
+
+  4. Now try deleting two words at a time using a count: Position the cursor on
+     the beginning of "two" and type «2dw»
+
+---> There are too many words in this sentence. {expect:There are words in this sentence.}
+---> There are words in this sentence. {expect:There are words in this sentence.}
+
+  4. On the line below move the curse behind the period. We don't want anyone
+     to read the second sentence, so move the cursor behind the period and hit
+     [«D»](D) to delete until the end of the line.
+
+---> The report will be done by Thursday. I hate my job so much... {expect:The report will be done by Thursday.}
+
+The operator [«d»](d) stands for "delete" and deletes according to a motion. If
+we type «dw» it will delete anything up until the beginning of the next word.
+We can use any other motion as well, so «de» will delete until including the
+last character of the current word, «dh» will delete the character left from the
+cursor and «dl» will delete the character under the cursor, similar to «x». Go
+ahead and try the «d» command with all the motions you have learned so far.
+
+# Lesson something.something: CHANGING TEXT
+
+** The «c» operator deletes text and places us in insert mode. **
+
+  1. Place the cursor at the beginning of a word you want to change.
+
+  2. Press «ce» or «cw» to change the word without affecting the surrounding whitespace.
+
+---> This lubw has a few wptfd that mrrf changing usf the change operator. {expect:This line has a few words that need changing using the change operator.}
+---> This line has a few words that need changing using the change operator. {expect:This line has a few words that need changing using the change operator.}
+
+  3. We want to change the last sentence. Move the cursor behind the first
+     period and press «C», enter to correct sentence and leave inster mode.
+
+---> The report will be done by Thursday. I hate my job so much... {expect:The report will be done by Thursday. This is the best job ever!}
+---> The report will be done by Thursday. This is the best job ever! {expect:The report will be done by Thursday. This is the best job ever!}
+
+The «c» operator is like deleting a piece of text with the «d» operator and
+then entering instert mode via «i». However, having an operator that does both
+in one step makes the action more repeatable.
+
+One word of caution: the «c» operator is somewhat of an anomaly when used with
+the «w» motion. Usually that motion would move to the beginning of the next
+word, but in this particular case it acts like the «e» motion. This is
+conscious design choice by the Vim developers, not a bug.
+
+# Lesson something.something: REPLACING TEXT
```

Comment:
I think it's odd that this doesn't have a subsection (like 1.8) like the other sections do.